docs/elements.md

```diff
@@ -275,6 +277,48 @@ Attribute | Type | Default | Description
 
 -----
 
+## `pl-dropdown` element
+
+Select the correct answer from a drop-down **select** menu list of potential answers. The potential options are listed in the inner HTML of a <pl-answer></pl-answer> element (ie. <pl-answer>Possible Answer 1</pl-answer>).
+
+#### Sample Element
+
+![](elements/pl-dropdown.png)
+
+**question.html**
+
+```html
+An <pl-dropdown sort="random" correct-answer="socrates">
+  <pl-answer correct="true">unexamined</pl-answer>
+  <pl-answer>examined</pl-answer>
+  <pl-answer>insatiable</pl-answer>
+</pl-dropdown> life is not worth living. <p></p>
+```
+
+**server.py**
+```python
+def generate(data):
+
+    # Correct Answers
+    data['correct_answers']['socrates'] = 'unexamined'
+```
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`correct-answer` | string | - | The key of the correct answer.
+`weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
+`sort` | string | fixed | Options are 'random', 'ascend', and 'descend', and 'fixed' for drop-down answers.
```

Comment:
Sort should probably be set to `random` by default as the top answers usually are "true" while the bottom answers are "false".

Reply:
I think that is strong reasoning, but it also worries me to have competing default behaviours when you write something in HTML like a drop-down and get a different result. I would let the option flag handle it in this case and let the question writer drive the question logic without too much "magic". It depends on how opinionated the PL framework is. If this is a strong opinion in PL, I can gladly implement this request.

Reply:
@nicknytko 

Here is an example of when only valid answers are submitted: 

![image](https://user-images.githubusercontent.com/7872295/85436812-7bedc000-b53e-11ea-83d4-d9c595f3aced.png)

Here is an example of when invalid answers are submitted with one correct answer: 

![image](https://user-images.githubusercontent.com/7872295/85436887-97f16180-b53e-11ea-9b3d-7ae720a86fba.png)

If there are any minor notes, please let me know. I think this is much closer to your UI example (except that the answers are within the sentence to give the answer context).

Reply:
The other elements (like `pl-multiple-choice`) use `random` as the default, so I think it would be good to be consistent and use `random` as default here.

Reply:
Done!